kettle/update.py

```diff
@@ -38,7 +38,7 @@ def call(cmd):
 
 
 def main():
-    call('time python3 make_db.py --buckets buckets.yaml --junit --threads 32')
+    call('time python3 make_db.py --buckets buckets.yaml --junit --threads 32 --buildlimit ' + os.getenv('BUILD_LIMIT', '0'))
```

Comment:
the defaulting logic should probably be where the flag is declared, instead of where it's used?

Reply:
default logic is there, but if the ENV is unset I think it will fail because the flag isn't an integer. would just be `TypeError: can only concatenate str (not "NoneType") to str`

Reply:
Wouldn't 

```python3
default=int(os.getenv('BUILD_LIMIT', '0'))
``` 
at

https://github.com/kubernetes/test-infra/blob/1579b01bab959eb44dbae5e410ef16546c3d45ff/kettle/make_db.py#L352

do the trick? or am i missing something

not a big deal though

/lgtm
/hold

Reply:
Ah I see what you mean, attempting to get default from ENV inside made_db.py?

Reply:
done